app/api/admin_sales/utils.py

```diff
@@ -10,7 +10,7 @@ def status_summary(orders, status):
     dictionary
     """
     return {
-        'sales_total': sum([o.amount for o in orders if o.status == status]),
+        'sales_total': sum([o.amount for o in orders if o.status == status and o.amount is not None]),
```

Comment:
Amount should never be None, make it non nullable

Reply:
@iamareebjamal Okay sir, working on it